amundsen_application/static/js/fixtures/globalState.ts

```diff
@@ -30,7 +30,10 @@ const globalState: GlobalState = {
     bookmarksForUser: [],
   },
   feedback: {
-      sendState: SendingState.IDLE,
+    sendState: SendingState.IDLE,
+  },
+  notification: {
```

Comment:
nit: I'm not sure that `notification.requestIsOpen` is the right path for this value, since this isn't really a `notification` itself-- I would reserve for in-app notifications. This is a mechanism that triggers other users to receive notifications.

I suppose the `feedback` is the most similar pattern here, with the difference that `feedback` lives on every page and `requestData` only exists on `table_detail`. You _could_ make the argument that state should be organized on a per page basis (with the exception of global things like logged-in-user, feedback, and all-tags). In that case `requestIsOpen` would be added to the `table_detail` state (currently named `tableMetadata`).

Alternatively, if this `requestIsOpen` exists in multiple pages, perhaps it should be similar to `feedback` and have its own state branch at the root. Even then we could change this to something like 
```
modals: {
  requestTableMetadata: {
     isOpen: boolean;
     sendState: SendingState;
  },
  feedback: {
    isOpen: boolean;
    sendingState: SendingState;
  }
}
```

Reply:
I think it's worth rearranging this state for sure. Though strong preference for doing this after the project is complete, as well as not adding `isOpen` for feedback in Redux because it's not needed based on how that component is used. I think consistency is great but only when the functionality is needed.

Reply:
:+1: if that's the case I'm just going to leave the redux state as is